travis.sh

```diff
@@ -10,26 +10,14 @@ if [ -z ${MODEL:-} ] ; then
     MODEL=64
 fi
 
-test_rdmd() {
-    # run rdmd internal tests
-    rdmd -m$MODEL -main -unittest rdmd.d
-
-    # compile rdmd & testsuite
-    dmd -m$MODEL rdmd.d
-    dmd -m$MODEL rdmd_test.d
-
-    # run rdmd testsuite
-    ./rdmd_test
-}
-
 build_digger() {
     git clone --recursive https://github.com/CyberShadow/Digger "$DIGGER_DIR"
     (cd "$DIGGER_DIR" && rdmd --build-only -debug digger)
 }
 
 install_digger() {
     $DIGGER build --model=$MODEL "master"
-    export PATH=$PWD/result/bin:$PATH
+    export PATH=$PWD/work/result/bin:$PATH
```

Comment:
I'd suggest pinning Digger to a specific version (as with DScanner) to avoid the breaking changes in Digger's master from affecting tools.